### ISSUE TITLE
x/vgo: netrc failing to load on Windows, possibly in wrong place

### ISSUE THREAD

**Issue by rsc** [golang/go#24606]
On Windows $HOME/.netrc is the wrong path; at the least there's no $HOME.
We should find out what the canonical path is on Windows and use that.

See https://github.com/golang/go/issues/23955#issuecomment-369387933 by @doxxx.

**Comment by doxxx** [comment]
On Windows, the equivalent of $HOME is $USERPROFILE. I do also have a HOME env var defined, pointing at the same location as USERPROFILE. I've created a `_netrc` file in that location with these contents:

```
machine api.github.com
login doxxx
password TOKEN
```

I've verified using Windows Subsystem for Linux that `vgo` with a `$HOME/.netrc` containing the above does not encounter the Github rate limits even after multiple invocations of `vgo get -u` in quick succession.

**Comment by doxxx** [comment]
I just tried renaming the `_netrc` to `.netrc` in Windows and it appears to be working with the contents I described above. I had tried `.netrc` originally, but maybe I got the format wrong at the time?

To summarize, for this to work:

* `.netrc` must be created in the user's profile directory (e.g. `C:\Users\Gordon`)
* Line endings don't matter, both CRLF and LF work.
* There must be a HOME env var containing the path to the user's profile directory. It doesn't work if this is not defined.
* The `.netrc` must use your Github username for the `login` and your generated Github auth token for the `password`.
